### PR TITLE
Add judgment and Modify level3 cannot be a keyword

### PR DESCRIPTION
**Pull request by iwasjoker**
Add try/catch in JDBCExample.java

### CODE REVIEW

File: example/jdbc/src/main/java/org/apache/iotdb/JDBCExample.java [hunk 1]
```diff
@@ -32,9 +34,23 @@ public static void main(String[] args) throws ClassNotFoundException, SQLExcepti
...
+        statement.execute("CREATE TIMESERIES root.sg1.d1.s3 WITH DATATYPE=INT64, ENCODING=RLE, CompressionType=SNAPPY");
+      } catch (IoTDBSQLException e) {
+        System.out.println(e.getMessage());
+      }
+
+      try {
+        statement.execute("CREATE TIMESERIES root.sg1.d1.s2 WITH DATATYPE=INT64, ENCODING=RLE, CompressionType=SNAPPY");
+      } catch (IoTDBSQLException e) {
+        System.out.println(e.getMessage());
+      }
```
Comment (qiaojialin) [inline example/jdbc/src/main/java/org/apache/iotdb/JDBCExample.java:53]: Hi, I think one try-catch is enough. Catch the set storage group and create timeseries.

Comment (iwasjoker) [reply example/jdbc/src/main/java/org/apache/iotdb/JDBCExample.java]: Hi，The original one is better，I'm sorry I did some useless jobs

Comment (qiaojialin) [reply example/jdbc/src/main/java/org/apache/iotdb/JDBCExample.java]: Hi~ Actually, I mean add a try-catch for set storage group and create timeseries

File: example/jdbc/src/main/java/org/apache/iotdb/JDBCExample.java [hunk 2]
```diff
@@ -48,6 +50,8 @@ public static void main(String[] args) throws ClassNotFoundException, SQLExcepti
       outputResult(resultSet);
       resultSet = statement.executeQuery("select count(*) from root where time >= 1 and time <= 100 group by ([0, 100), 20ms, 20ms)");
       outputResult(resultSet);
+    }catch (IoTDBSQLException e){
```
Comment (jixuan1989) [inline example/jdbc/src/main/java/org/apache/iotdb/JDBCExample.java:53]: Hi, leave a blank space between "}" and "catch"

File: example/jdbc/src/main/java/org/apache/iotdb/JDBCExample.java [hunk 3]
```diff
@@ -30,11 +32,11 @@
...
     Class.forName("org.apache.iotdb.jdbc.IoTDBDriver");
     try (Connection connection = DriverManager.getConnection("jdbc:iotdb://127.0.0.1:6667/", "root", "root");
-         Statement statement = connection.createStatement()) {
+      Statement statement = connection.createStatement()) {
       statement.execute("SET STORAGE GROUP TO root.sg1");
-      statement.execute("CREATE TIMESERIES root.sg1.d1.s1 WITH DATATYPE=INT64, ENCODING=RLE");
-      statement.execute("CREATE TIMESERIES root.sg1.d1.s2 WITH DATATYPE=INT64, ENCODING=RLE");
-      statement.execute("CREATE TIMESERIES root.sg1.d1.s3 WITH DATATYPE=INT64, ENCODING=RLE");
+      statement.execute("CREATE TIMESERIES root.sg1.d1.s1 WITH DATATYPE=INT64, ENCODING=RLE, CompressionType=SNAPPY");
+      statement.execute("CREATE TIMESERIES root.sg1.d1.s3 WITH DATATYPE=INT64, ENCODING=RLE, CompressionType=SNAPPY");
```
Comment (jixuan1989) [inline example/jdbc/src/main/java/org/apache/iotdb/JDBCExample.java:38]: why exchange "s3" and "s2"?

**Comment by jixuan1989** [comment]
Hi @qiaojialin , how do you think.

@iwasjoker , so "device" can not appear in the path?

**Comment by iwasjoker** [comment]
> Hi @qiaojialin , how do you think.
> 
> @iwasjoker , so "device" can not appear in the path?

**Pull request merged**